epictrack-web/cypress/support/utils.ts

```diff
@@ -0,0 +1,71 @@
+import { Staff } from "models/staff";
+
+export function createMockMasterContext(defaultItem: any, _data?: any) {
+  return {
+    item: defaultItem,
+    setFormId: cy.stub(),
+    setTitle: cy.stub(),
+    setId: cy.stub(),
+    onSave: cy.stub(),
+    title: "",
+    data: [..._data],
```

Comment:
Can se keep the "setupIntercepts" in utils.ts and rest of them to another file in the same folder called "common.ts"